### PR TITLE
Add Publish Requirements

### PR DESCRIPTION
**Pull request by Farhad-Farahi**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -20,4 +20,38 @@ kotlin.code.style=official
...
 # thereby reducing the size of the R class for that library
-android.nonTransitiveRClass=true
\ No newline at end of file
+android.nonTransitiveRClass=true
+
+SONATYPE_HOST=S01
+RELEASE_SIGNING_ENABLED=true
+
+
+GROUP=cab.snapp.blackBox
```
Comment (HadiHabibza) [inline gradle.properties:29]: it should be io.github.snapp-incubator

File: gradle.properties [hunk 2]
```diff
@@ -20,4 +20,38 @@ kotlin.code.style=official
...
+POM_ARTIFACT_ID=blackBox
+VERSION_NAME=1.0.0
+
+POM_NAME=sherlock
```
Comment (HadiHabibza) [inline gradle.properties:33]: why sherlock? isn't it poaro?

File: gradle.properties (hunk 1 above)
Comment (Farhad-Farahi) [reply gradle.properties]: Thanks @HadiHabibza jan, Done

File: gradle.properties (hunk 2 above)
Comment (Farhad-Farahi) [reply gradle.properties]: Thanks @HadiHabibza jan, Done

**Pull request merged**